File: Data Structures & Algorithms/max-consecutive-ones/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMaxConsecutiveOnes(nums) {
        let maxCons = 0;
        let counter = 0;
        for(let num of nums){
            if(num !== 1){
                counter = 0;
            }else{
                counter++;
                if (counter > maxCons){
                    maxCons = counter;
                };
            }
        }
        return maxCons;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include <unordered_map>
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int, int> map; 
        for(int num : nums){
            if(map.find(num) != map.end()){
                return true;
            }
            map[num] = 1;
        }
        return false;
    }
};